plasmapy/particles/atomic.py

```diff
@@ -1122,7 +1124,7 @@ def periodic_table_category(argument: Union[str, Integral]) -> str:
             "symbol, or an integer representing its atomic number."
         )
     symbol = atomic_symbol(argument)
-    category = _elements[symbol]["category"]
+    category = _data_about_elements[symbol]["category"]
```

Comment:
Perhaps rename to `_element_data_dict`, `_isotope_data_dict`, `_special_particle_data_dict`?